DEVELOPERS.md

```diff
@@ -0,0 +1,76 @@
+# Cicero Word Add-in Development Guide
+
+## ❗ Accord Project Development Guide ❗
+We'd love for you to help develop improvements to Cicero Word Add-in! Please refer to the [Accord Project Development guidelines][apdev] we'd like you to follow.
+
+## Cicero Word Add-in Specific Information
```

Comment:
All the information below is already in the README so no point repeating it. I would suggest to keep this but delete the instructions in the README but provide a link there to this markdown file.

Reply:
> so no point repeating it.

Yeah 😄 

>  I would suggest to keep this but delete the instructions in the README but provide a link there to this markdown file.

Sure I will do these changes soon.